### PR TITLE
Fix: add missing parameters maxResponses to discussion api requests

### PR DESCRIPTION
**Pull request by TBonnin**
Adding maxResponses query string parameter to /discussion/... request

### CODE REVIEW

**Comment by gidsg** [comment]
:+1: :gem: (if this works as expected on CODE) It should *I think* have a view more comments button if there are hidden comments. 

screen shot 2016-01-13 at 13 26 38 2

**Comment by TBonnin** [comment]
It looks like this on my local: It now shows a "+ Show X more replies" button below the 3 first replies when the `collapsed` option is selected

picture 1

**Comment by TBonnin** [comment]
I think refactoring the way query string parameters are handled in this (https://github.com/guardian/frontend/blob/tbonnin-fix-collapsed-comments/discussion/app/discussion/DiscussionApi.scala)[DiscussionAPI file] would be nice. :)

**Comment by TBonnin** [comment]
cc @jennysivapalan Can I have a review when you have some time? Thanks

File: discussion/app/discussion/DiscussionApi.scala [hunk 1]
```diff
@@ -71,7 +72,8 @@ trait DiscussionApi extends Http with ExecutionContexts with Logging {
       "orderBy" -> params.orderBy,
       "displayThreaded" -> (params.displayThreaded match {
         case false => "false"
-        case _ => None}))
+        case _ => None})
+    )
```
Comment (TBonnin) [inline discussion/app/discussion/DiscussionApi.scala]: Only formatting for readibility

File: discussion/test/discussion/DiscussionApiTest.scala [hunk 2]
```diff
@@ -56,7 +56,6 @@ import scala.language.postfixOps
       page = "1",
       pageSize = "50",
       maxResponses = None,
-      topComments = false,
```
Comment (TBonnin) [inline discussion/test/discussion/DiscussionApiTest.scala]: Unused param, so I am removing it.

File: discussion/test/discussion/DiscussionApiTest.scala [hunk 3]
```diff
@@ -68,8 +67,6 @@ import scala.language.postfixOps
       orderBy = "newest",
       page = "1",
       pageSize = "50",
-      maxResponses = None,
-      topComments = false,
```
Comment (TBonnin) [inline discussion/test/discussion/DiscussionApiTest.scala]: Unused params, so I am removing them.

**Comment by jennysivapalan** [comment]
:+1: :speech_balloon:

**Pull request merged**